.github/workflows/ci.yaml

```
@@ -375,6 +375,23 @@ jobs:
             - name: Check
               run: cargo check --target=${{matrix.target}} -p printerdemo_mcu --no-default-features --features=mcu-board-support/${{matrix.feature}} --release
 
+    # test to compile the mcu backend for the arm target (no_std) using embassy
+    mcu-embassy:
+        env:
+            SLINT_FONT_SIZES: 8,11,10,12,13,14,15,16,18,20,22,24,32
+            RUSTFLAGS: --cfg slint_int_coord -D warnings
+            CARGO_PROFILE_DEV_DEBUG: 0
+            CARGO_PROFILE_RELEASE_OPT_LEVEL: s
+        runs-on: ubuntu-22.04
+        steps:
+            - uses: actions/checkout@v4
+            - uses: ./.github/actions/setup-rust
+              with:
+                  target: thumbv8m.main-none-eabihf            
```

The autofix CI checks that there is no trailing space.  
This line contains trailling spaces. So the CI removes it, but  somehow autofix can't change files in the .github directory.

```suggestion
                  target: thumbv8m.main-none-eabihf
```